### PR TITLE
Fixing package.json to address the problem of truffle failure

### PR DESCRIPTION
**Pull request by vishwa-raman**
There was a bindings dependency that was not found when we tried to compile and test. Missing dependencies in package.json. With these changes I checked that the test passes in the c-k container.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,23 +1,23 @@
...
-    "web3c",
-    "ekiden"
-  ],
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/oasislabs/oasis-box.git"
+    "json-loader": "^0.5.4",
+    "oasis-compiler": "^1.0.0",
+    "style-loader": "^0.13.1",
+    "truffle": "^4.1.14",
```
Comment (armaniferrante) [inline package.json]: Shouldn't we be using our own version of truffle?

Comment (armaniferrante) [inline package.json]: It seems wrong to include truffle here.

Comment (vishwa-raman) [reply package.json]: Removed. It is wrong.

**Comment by vishwa-raman** [comment]
Tested as follows in the contract kit container.

Pull existing erc20-rust master
npm install .
truffle test --network development

Pull PR branch erc20-rust
npm install .
truffer test --network development

The former errors out not finding bindings. The latter produces -

```
oot@5a73a5737a77:~/erc20-rust# truffle test --network development
Building contracts for Oasis
   Compiling erc20-token...
   Compiling Migrations.sol...


  Contract: TokenContract
    ✓ should have a balance of 100
    ✓ should transfer balance of 16 yielding (84, 16) (140ms)


  2 passing (182ms)
```

**Pull request merged**